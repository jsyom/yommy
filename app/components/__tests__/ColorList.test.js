import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ColorList from '../ColorList';

test('ColorList component should render as expected', () => {
	const component = shallow(<ColorList />);
	const tree = toJson(component);
	expect(tree).toMatchSnapshot();
});
