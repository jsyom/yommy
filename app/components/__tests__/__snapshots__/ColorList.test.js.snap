// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorList component should render as expected 1`] = `
<div
  className="colorlist-container"
>
  <div
    className="error-message"
  >
    <h3 />
  </div>
  <div
    className="palette-container"
  />
</div>
`;
